packages/mask/src/components/shared/SelectRecipients/SelectRecipientsDialog.tsx

```diff
@@ -146,11 +143,11 @@ export function SelectRecipientsDialogUI(props: SelectRecipientsDialogUIProps) {
                     value={search}
                     onKeyUp={(e) => {
                         if (e.keyCode === 13) {
```

Comment:
`keyCode` is deprecated, see https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

```suggestion
                        if (e.code !== 'Enter') return
```

Reply:
done